### DISCUSSION TITLE
Blog Sticky Till The End Module

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#766] [General]
# Blog Sticky Till The End Module

By default, the sidebar and ToC (table of contents) will sticky until reach the footer, with this module, they can be sticky all the time.

https://hbstack.dev/modules/blog/sticky-till-the-end/

**Comment by LITUATUI** [comment]
Thanks for the module!

Can we configure it to stop when reaching an element that isn't the footer? In WordPress I remember that I once used a plugin that had this option.

For example, I use CSS to don't display the footer (display: none), so I want the sidebar to stop at body-end hook (where I have a banner).

**Reply by razonyang** [reply]
No, but you can override `layouts/partials/hb/modules/footer/index.html`, and put your custom content there.

```
<div class="hb-footer">
... 
</div>
```

Please note that you're using this on your own risks, no guaranteed.

**Reply by LITUATUI** [reply]
Thanks, it works, but in my case the correct path is:

```
layouts/partials/hb/modules/footer/index.html

**Reply by razonyang** [reply]
My mistake, corrected.
